initial_setup/product.py

```diff
@@ -1,90 +1,50 @@
 """Module providing information about the installed product."""
 import logging
 
-from pyanaconda.localization import find_best_locale_match
-from pyanaconda.core.constants import DEFAULT_LANG
-import os
-import glob
-
-RELEASE_STRING_FILE = "/etc/os-release"
-LICENSE_FILE_GLOB = "/usr/share/redhat-release*/EULA*"
```

Comment:
I could find only `/usr/share/redhat-release/EULA` on the installed system.

Reply:
Yep, as far as I can tell this is the only file used these days, so should be correct. :)